### ISSUE TITLE
[Bug]: Extensions installation error: "remote-https' is not a git command"

### ISSUE THREAD

**Issue by BigZampano** [brkirch/stable-diffusion-webui#4]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What happened?

I already had an installation of automatic1111-webui on my Mac Mini which was running quite nicely until a week ago, when an update messed up my Gradio interface a little... images showed while generating, but the finished image just vanished from the preview window... images were still saved and accessible from image browser...

I then tried your install, which went easy enough, everything was working just fine...

I then decided to switch my stuff to your install, but when trying to install extensions (e.g. image browser), I get an error:
'remote-https' is not a git command'

Trying to install by URL gives the same error...

### Steps to reproduce the problem

1. Go to "extensions"
2. Try to install an extension (either by clicking "install" or by pasting the URL)
3. error

### What should have happened?

Extension should install

### Commit where the problem happens

4b159292

### What platforms do you use to access the UI ?

MacOS

### What browsers do you use to access the UI ?

Google Chrome

### Command Line Arguments

```Shell
No
```

### List of extensions

No, because they don't install...

### Console logs

```Shell
Error completing request
Arguments: ('https://github.com/AlUlkesh/stable-diffusion-webui-images-browser.git', ['ads', 'localization', 'installed'], 0) {}
Traceback (most recent call last):
  File "/Volumes/AI-Models/stable-diffusion-webui-mac/modules/call_queue.py", line 56, in f
    res = list(func(*args, **kwargs))
  File "/Volumes/AI-Models/stable-diffusion-webui-mac/modules/ui_extensions.py", line 171, in install_extension_from_index
    ext_table, message = install_extension_from_url(None, url)
  File "/Volumes/AI-Models/stable-diffusion-webui-mac/modules/ui_extensions.py", line 145, in install_extension_from_url
    repo = git.Repo.clone_from(url, tmpdir)
  File "/Volumes/AI-Models/stable-diffusion-webui-mac/python/3.10.10/lib/python3.10/site-packages/git/repo/base.py", line 1308, in clone_from
    return cls._clone(
  File "/Volumes/AI-Models/stable-diffusion-webui-mac/python/3.10.10/lib/python3.10/site-packages/git/repo/base.py", line 1219, in _clone
    finalize_process(proc, stderr=stderr)
  File "/Volumes/AI-Models/stable-diffusion-webui-mac/python/3.10.10/lib/python3.10/site-packages/git/util.py", line 419, in finalize_process
    proc.wait(**kwargs)
  File "/Volumes/AI-Models/stable-diffusion-webui-mac/python/3.10.10/lib/python3.10/site-packages/git/cmd.py", line 604, in wait
    raise GitCommandError(remove_password_if_present(self.args), status, errstr)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  cmdline: git clone -v -- https://github.com/AlUlkesh/stable-diffusion-webui-images-browser.git /Volumes/AI-Models/stable-diffusion-webui-mac/tmp/stable-diffusion-webui-images-browser
  stderr: 'Cloning into '/Volumes/AI-Models/stable-diffusion-webui-mac/tmp/stable-diffusion-webui-images-browser'...
warning: templates not found in /opt/homebrew/Cellar/git/2.40.0/share/git-core/templates
git: 'remote-https' is not a git command. See 'git --help'.
```

**Comment by jrittvo** [comment]
Until this gets fixed, you can copy over extensions from your old version of Auto1111, if you have one, or you can try downloading a zip of the particular git repo and extract it into your Auto1111 extension folder,  Near the top right of each git repo page, there is a green "Code" button that has "Download ZIP" as a choice.

**Comment by BigZampano** [comment]
Thanks for the suggestion, but:
I did that the first time (copied models + extensions), but this messed everything up on first launch and I had to delete the installation... that's why I decided to go the hard way and install every extensions from scratch, which then didn't work...
But no problem, I think I'll stick to my old install, somehow I was able to revert to an older Gradio version 3.16.2 and it seems to work again...
But still, great work for making installation easy for non-tech people..!

**Comment by sonluong1987** [comment]
I believe it's caused by line 50 inside webui-user.sh: 

`export PATH="$PWD/git/bin:$PATH"
`

This installation tries to use a git binary which was probably not compiled with libcurl. Commenting out that line and let webui use your system git fixed the problem for me.

**Comment by jrittvo** [comment]
Something is strange about the git that gets installed with this app version.  Mine has a folder `git/libexec/git-core` that is 480 MB.

**Comment by brkirch** [comment]
The issue with loading extensions is fixed with [20230411_experimental](https://github.com/brkirch/stable-diffusion-webui/releases/tag/20230411_experimental). The next stable build (which should be released within the next few days) will have this fixed as well.

> Something is strange about the git that gets installed with this app version. Mine has a folder `git/libexec/git-core` that is 480 MB.

At some point I must have done a copy that changed the symbolic links to regular files, although actually I'm not completely sure exactly when or how that happened. I'll also have that fixed in the next stable build.

**Comment by jrittvo** [comment]
I just did a clean install of your 0411 build.  I need to link my extensions, models, etc. and then I'll give it a try tomorrow.  I've been "merging" most of your commits for this from the past 2 weeks into my standard Auto1111 installation, so it will be interesting to see if there are any further performance differences.  The tomesd is a nice speed up. I used the pull request at Automatic1111 to add that along with your script mods.  I built your torch version last night and put it in there too, but it got tripped up on calls to the torch nn distributed modules.  I either need to try again and change something in the build options, add an argument to the startup script, or comment out things, so it doesn't try to load that stuff, but with this new full app out, I'll skip that effort for now.  Thanks for sticking with this.
